client/client.go

```diff
@@ -335,8 +335,11 @@ type Client struct {
 	// partitions is used for managing cpuset partitioning on linux systems
 	partitions cgroupslib.Partition
 
-	// widmgr retrieves workload identities
-	widmgr *widmgr.WIDMgr
+	// widsigner signs workload identities
+	widsigner widmgr.IdentitySigner
+
+	// widmgr manages workload identities
+	widmgr widmgr.IdentityManager
```

Comment:
```suggestion
```

I think we can remove this one since each allorunner has a `widmgr` now?

Reply:
good point, fixed in [d2fa276](https://github.com/hashicorp/nomad/pull/18431/commits/d2fa276e186e900acea769d7129238cb22fe78ee)